Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,43 @@
+{
+	"@context":{
+		"vocab":"http://temperaturesystems.com/Products/#",
+		"Class01":"vocab:Class01",
+		"Sensor029":"Class01:Sensor029",
+		"Computer101":"Class01:Computer101",
+		"Measurement8401":"Sensor029:Measurement8401",

```

Comment:
The definitions are a bit confusing. But don't worry, we'll see how to better define them in our ontology session :-)

Reply:
Sorry,  I did my best but I still find it a bit confusing

Reply:
Not worries ;-)